RInAppMessaging/Classes/Protocols/HttpRequestable.swift

```diff
@@ -32,9 +35,17 @@ internal protocol HttpRequestable {
                            addtionalHeaders: [HeaderAttribute]?,
                            completion: @escaping (_ result: RequestResult) -> Void)
 
+    func buildURLRequest(url: URL) -> Result<URLRequest, Error>
+
     func buildHttpBody(with parameters: [String: Any]?) -> Result<Data, Error>
 }
 
+extension HttpRequestable {
+    func buildURLRequest(url: URL) -> Result<URLRequest, Error> {
+        .success(URLRequest(url: url))
```

Comment:
Is this always returning success? It can't fail?

Reply:
It can't fail. It is the default implementation of `buildURLRequest`.